devops/ansible/easymode-setup.py

```diff
@@ -0,0 +1,35 @@
+from GirderClient import *
```

Comment:
We now have the `girder_client` package in pip, we should probably port to using that at some point.